reports/m1_proposal.md

```diff
@@ -0,0 +1,20 @@
+# Milestone 1 - Dashboard proposal(Vancouver Crime Tracker)
+
+## 1. Motivation and purpose
+
+
+## 2. Description of the data
+
+
+
+## 3. Research questions and usage scenarios
+
+Adam Palmer is a Chief Constable at the Vancouver Police Department in British Columbia and he has come to understand that many of the citizens living in Vancouver feel extremely unsafe stepping foot outside their homes. Therefore, he wants to understand what is causing this perception, and how he can effectively manage the resources at his disposal to reduce the crimes in Vancouver. He wishes to be able to take a glance at a holistic overview of the crime data and determine which types of crimes should be targetted and what times he should be allocating most of his patrols.
+
+When Adam logs into our "Crime Tracker app", he will see an overview of the variables available in our dataset, according to the number of crimes being committed for different categories, the times at which they were committed and the neighbourhoods in which they were commited. He can then filter-out different types of crimes, or the neighbourhoods or time, to make comparisons to determine which areas of resource allocation would be most effective for crime reduction. When he does so, Adam may perhaps notice that there seems to be a greater count of "robberies/snatchings" being committed at "night" in "Hastings"
+
+Based on his findings from using our app, Adam hypothesizes that in order to reduce snatch/grab/run crimes, a reduction in which would make the population of Vancouver feel safe, he can allocate greater resources for patrolling in Hastings at night, in an attempt to reduce crimes there.
```

Comment:
Hi Waleed! Thanks for the write up. I think it looks great overall! Just a very minor recommendation from my side :)